src/Illuminate/Database/Eloquent/Concerns/GuardsAttributes.php

```diff
@@ -60,12 +61,12 @@ public function fillable(array $fillable)
     /**
      * Merge new fillable attributes with existing fillable attributes on the model.
      *
-     * @param  array  $fillable
+     * @param  array|string  $fillable
      * @return $this
      */
-    public function mergeFillable(array $fillable)
+    public function mergeFillable($fillable)
     {
-        $this->fillable = array_merge($this->fillable, $fillable);
+        $this->fillable = array_merge($this->fillable, Arr::wrap($fillable));
```

Comment:
Using Arr::wrap(..) to wrap the inserted fillable in an array as it should be if it's not an array before.